src/Reflection/ReflectionObject.php

```diff
@@ -86,8 +86,14 @@ public static function createFromInstance($object)
             throw new \InvalidArgumentException('Can only create from an instance of an object');
         }
 
+        $className = get_class($object);
+
+        if (strpos($className, 'class@anonymous') === 0) {
+            throw new \LogicException('Not implemented - Unable to locate anonymous class');
```

Comment:
Is this the same exception thrown by internals? Also, I think we can actually locate them, compared to internal reflection...